db/migrate/20200206144913_add_membership_type_ref_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddMembershipTypeRefToUsers < ActiveRecord::Migration[5.2]
+  def change
+    add_reference :users, :membership_type, foreign_key: true
```

Comment:
Do I need this on the users table?

Reply:
@pollygee  It kind of depends. If we put the membership type on the user's table, then we have to put this there, along with the expiration date and any other data that relates to the membership. A better alternative might be to have a memberships table (which would have `membership_type`, `user_id`, `expiration_date`, etc.). 

The remaining question would be whether the user `has_one` membership or `has_many` memberships (in the latter case we would add a new membership whenever a user joins or renews. The latter retains all membership history while being slightly more complicated to code.

Reply:
I like the idea of having a memberships table.  Then we can see a history of the memberships as well.